### ISSUE TITLE
[Builder] use last maximum_color from nanoemoji to fix color fonts

### ISSUE THREAD

**Issue by RosaWagner** [googlefonts/gftools#656]
At the moment after the Builder had fixed the color font, the SVG table is still missing. I noticed Cosimo was re-running maximum_color on color fonts to fix that, using https://github.com/googlefonts/nanoemoji/releases/tag/v0.15.1

**Comment by m4rc1e** [comment]
Just spoke to Cosimo and his response is:

> I think fontbakery enforces that if a font has colrv0, it should not also have svg
we assumed (wrongly) that COLRv0 would be sufficient and I think Rod prefers to keep it simple so we said, if it has COLRv1 then should also have SVG; if it has COLRv0 it should not
but turns out adobe apps don't work with COLRv0 even
adding SVG table to static COLR v0 (or v1) fonts sounds sensible to me
there's also the problem of encoding all this logic on the serving side of things, to deliver the right subset given the user agent, hence why we went for KISS
anyway you could try bring this up to Rod and see what he thinks
for browsers at least, SVG table is only ever needed as fallback for unsupported COLRv1

@rsheeter any problems if we add SVG tables to static colr v0 fonts?

**Comment by rsheeter** [comment]
> any problems if we add SVG tables to static colr v0 fonts?

No, but we need to coordinate with oncall as I believe we will need internal code changes as well. Currently I believe the serving would do the right thing but our deployment process would reject COLRv0+SVG as a combination that shouldn't exist. @chrissimpkins I wonder if you can help be the bridge between worlds here?

**Comment by rsheeter** [comment]
(so please don't start adding svg to colr v0 quite yet)

**Comment by chrissimpkins** [comment]
I spoke with Rod about this.  We're going to test with one family before we do this more broadly.  I connected with Marc and Rosalie in a separate channel about the plan.
